ComfortZoneApp/app/src/main/java/com/example/comfortzone/flight/ui/CityMapViewFragment.java

```diff
@@ -31,13 +34,15 @@
 public class CityMapViewFragment extends Fragment implements OnMapReadyCallback, UpdateCityListCallback {
 
     public static final String TAG = "CityMapViewFragment";
-    public static final int MAX_CITIES = 20;
+    public static final int TOTAL_CITIES = 198;
```

Comment:
could you get this info from city data instead when you load it? that way it's scalable